zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -109,21 +109,14 @@ pub(super) struct CandidateSet<S> {
     pub(super) address_book: Arc<std::sync::Mutex<AddressBook>>,
     pub(super) peer_service: S,
     next_peer_min_wait: Sleep,
+    next_update_time: Instant,
```

Comment:
Naming suggestion:
1. It's easy to confuse "next peer", "new peers", "GetAddr" and "update"
2. next_update_time is the minimum next update time, not the always next update time

So let's rename them to:
* min_next_handshake
* min_next_crawl

Let's also add a short doc comment for both of the time fields, explaining that one of them adds new peers to the peer set using GetAddr, and the other returns the next handshake peer. We should also mention the function each time controls.

Reply:
(We can replace the Sleep with an Instant in a follow-up PR, and update its test to use tokio::time::advance.)